index.html

```diff
@@ -218,77 +237,159 @@ <h2>Jeszcze więcej</h2>
 
 <!-- !!!START FEATURE FORM - MACIEK  -->
 
-<section id="sign-in" class="subscribe">
+<section
```

Comment:
jeżeli decydujecie się na taki zapis to również pamiętajcie aby ">" znajdował się w nowej lini, w ten sposób dopisująć kolejny atrybut nie trzeba modyfikować lini z ostatnim